### PR TITLE
Update top-scoring-industry-tests.md

### LINKED ISSUE #12120: Please add the AV-TEST 2022 results
Can we add the 2022 test from AV-TEST 
https://techcommunity.microsoft.com/t5/microsoft-defender-for-endpoint/microsoft-awarded-best-advanced-protection-for-corporate-and/ba-p/3765332
https://www.av-test.org/en/news/av-test-award-2022-for-microsoft/

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 0665c47f-53f5-bd90-bb22-c99650f2c0ff
* Version Independent ID: a832271a-63b1-71fe-043e-0e3910bb61e0
* Content: [Top scoring in industry tests - Microsoft 365 Defender](https://learn.microsoft.com/en-us/microsoft-365/security/defender/top-scoring-industry-tests?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender/top-scoring-industry-tests.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender/top-scoring-industry-tests.md)
* Service: **microsoft-365-security**
* Sub-service: **m365d**
* GitHub Login: @Dansimp
* Microsoft Alias: **dansimp**

### LINKED ISSUE #12121: SE Labs test result missing (2023)
Can we add "Microsoft Defender for Office 365 named Best Email Security Service of 2023 by SE Labs" https://www.microsoft.com/en-us/security/blog/2023/02/21/microsoft-defender-for-office-365-named-best-email-security-service-of-2023-by-se-labs/

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 0665c47f-53f5-bd90-bb22-c99650f2c0ff
* Version Independent ID: a832271a-63b1-71fe-043e-0e3910bb61e0
* Content: [Top scoring in industry tests - Microsoft 365 Defender](https://learn.microsoft.com/en-us/microsoft-365/security/defender/top-scoring-industry-tests?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender/top-scoring-industry-tests.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender/top-scoring-industry-tests.md)
* Service: **microsoft-365-security**
* Sub-service: **m365d**
* GitHub Login: @Dansimp
* Microsoft Alias: **dansimp**

### PR DESCRIPTION
**Pull request by msbemba**
Added latest Av test award

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12120

Added SE Award

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12121

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender/top-scoring-industry-tests.md [hunk 1]
```diff
@@ -52,6 +52,8 @@ Microsoft Defender Antivirus is the [next generation protection](https://www.you
 
 The AV-TEST Product Review and Certification Report tests on three categories: protection, performance, and usability. The following scores are for the Protection category that has two scores: Real-World Testing and the AV-TEST reference set (known as "Prevalent Malware").
 
+- 2022 AV-TEST Award – for tested IT Security :  [Best Advanced Protection for Consumer Users and Best Advanced Protection](https://www.av-test.org/en/news/av-test-award-2022-for-microsoft/)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender/top-scoring-industry-tests.md]:
```suggestion
- 2022 AV-TEST Award – for tested IT Security:  [Best Advanced Protection for Consumer Users and Best Advanced Protection](https://www.av-test.org/en/news/av-test-award-2022-for-microsoft/)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Pull request merged**